pkgs/dash_analytics/CHANGELOG.md

```diff
@@ -0,0 +1,3 @@
+## 1.0.0
```

Comment:
I'd have this match the pubspec version. The actual version may not matter greatly since it will only be in use by a few tools. You could do `0.1.0` initially - until you think any APIs changes are less likely - or start w/ `1.0.0` and just plan to rev if you make any breaking api changes.

Reply:
I'll mention again that we don't actually need to publish this package until we're ready to use it in a published package. Given that the `analysis_server` package isn't published, we might have some time before publishing is necessary.